packages/react/src/components/ListBox/ListBoxField.js

```diff
@@ -24,8 +24,8 @@ const ListBoxField = ({ children, id, ...rest }) => (
     role="combobox"
     aria-haspopup="listbox"
     aria-expanded={rest[`aria-expanded`]}
-    aria-owns={`${id}__menu`}
-    aria-controls={`${id}__menu`}
+    aria-owns={(rest[`aria-expanded`] && `${id}__menu`) || null}
```

Comment:
Could we add some tests for these conditionals just to verify what needs to be happening?